packages/babel-generator/test/index.js

```diff
@@ -489,7 +489,7 @@ suites.forEach(function(testSuite) {
             const actualAst = parse(actualCode, {
               filename: actual.loc,
               plugins: task.options.plugins || [],
-              strictMode: false,
```

Comment:
This was not really correct as `sourceType:"module"` should always be in strict mode.